### PR TITLE
Feature/multipage update

### PR DESCRIPTION
**Pull request by Phizzard**

### CODE REVIEW

File: gatsby-node.js [hunk 1]
```diff
@@ -1,11 +1,136 @@
...
+        username: process.env.PODBEAN_API_TOKEN,
+        password: process.env.PODBEAN_API_SECRET,
+      },
+      data: {
+        grant_type: "client_credentials",
+      },
+    })
+
+    const { data } = await axios.get(
+      `https://api.podbean.com/v1/episodes?access_token=${auth.data.access_token}`
```
Comment (brewsterbhg) [inline gatsby-node.js]: `api.podbean.com` could be in a const called BASE_URL but maybe I'm just BEING PICKY

File: src/pages/index.js [hunk 2]
```diff
@@ -1,43 +1,26 @@
...
-        scrolling="no"
-        width="100%"
-        height="432"
-        allowfullscreen=""
-      ></iframe>
-    </>
-  )
+
+const IndexPage = ({ data }) => {
+  const { fridayNightDeployEpisode } = data
```
Comment (brewsterbhg) [inline src/pages/index.js]: It's a pretty long prop name but that doesn't matter that much 😜

File: gatsby-node.js (hunk 1 above)
Comment (Phizzard) [reply gatsby-node.js]: I thought about this too at some point but must have forgotten about it later, I'll throw it into a constant

File: gatsby-node.js [hunk 3]
```diff
@@ -1,11 +1,136 @@
...
+      ""
+    )
+
+    episode.publish_time_formatted = format(
+      new Date(episode.publish_time * 1000),
+      "MMM do, yyy"
+    )
+
+    // This definitely wasn't from a stack overflow post
+    let sec = episode.duration
```
[the remark this answers is not in the document]
Comment (Phizzard) [reply gatsby-node.js]: Yeah good idea, It's an obnoxious snippet I ran with from being frustrated with trying to generate this duration length so I can clean it up a little bit

File: src/pages/index.js (hunk 2 above)
Comment (Phizzard) [reply src/pages/index.js]: GOOD CAUSE I DON'T CARE

**Pull request merged**